Automattic/jetpack
# Extensions: drop i18n wrapper

Closes: #11656 

#### Changes proposed in this Pull Request:

- Drop the now-obsolete i18n helper in `extensions/utils`
- Change `__, _n, _x, _nx()` imports to import directly from `@wordpress/i18n`
- Add `jetpack` textdomain as last argument to all those functions

#### Testing instructions:

- Verify that blocks still work
- (Not sure if we currently expect translations to work? They might live in a different place than they will once this is merged and used for string extraction.)

#### Proposed changelog entry for your changes:

Blocks: Drop i18n wrapper, use `@wordpress/i18n` directly.
